commands.js

```diff
@@ -1701,6 +1706,7 @@ var commands = exports.commands = {
 			});
 		});
 	},
+	updateserverhelp: ["/updateserver - Updates the server to a newer version. Requires: Console Access"],
```

Comment:
And yes, this documentation is useless unless it explains exactly how it updates the server.